app/controllers/assignments_controller.rb

```diff
@@ -582,6 +586,10 @@ def download_config_files
         yml_criteria = assignment.criteria.reduce({}) { |a, b| a.merge b.to_yml }
         f.write yml_criteria.to_yaml
       end
+      zipfile.get_output_stream(CONFIG_FILES[:annotations]) do |f|
+        f.write convert_to_yml(assignment.annotation_categories)
```

Comment:
I should have commented this in a previous PR. But if you're going to use the `convert_to_yml` function from the annotation_categories_helper.rb file, please change the name to make it clear that this isn't a generic function to convert some thing to yml. Or even better, put it in the AnnotationCategory model so you can call it like: `AnnotationCategory.convert_to_yml(assignment.annotation_categories)`